src/error.rs

```diff
@@ -126,6 +126,9 @@ pub enum LoadMetadata {
 
     /// Malformed Regex in Metadata XML database
     #[error("Malformed Regex: {0}")]
-    Regex(#[from] regex::Error),
+	Regex(#[from] regex::Error),
```

Comment:
Something funky going on here with spaces/tabs.

Reply:
that was wired, fixed